migrator/zenodo_rdm_migrator/actions/users.py

```diff
@@ -0,0 +1,152 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# Invenio-RDM-Migrator is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Invenio RDM migration users actions module."""
+
+
+from invenio_rdm_migrator.actions import TransformAction
+from invenio_rdm_migrator.load.postgresql.transactions.operations import OperationType
+from invenio_rdm_migrator.streams.actions import (
+    UserDeactivationAction,
+    UserEditAction,
+    UserRegistrationAction,
+)
+from invenio_rdm_migrator.transform import IdentityTransform
+
+from ..transform.entries.users import ZenodoUserEntry
+
+
+class ZenodoUserRegistrationAction(TransformAction):
```

Comment:
```suggestion
class UserRegistrationAction(TransformAction):
```

nit: to keep class names small, and maybe rely on namespacing?

Reply:
Done, I've renamed classes using `Load` as prefix for RDM-Migrator ones. However, this is inconsistent with transformers since we have been using `ZenodoRecordTransform`, `ZenodoUserTransform`, etc.

Reply:
Agree... Maybe if we added some folder-level prefixing it would help, i.e. `zenodo_rdm_migrator/actions/transform/users.py`. Then we could just name the class `UserRegistration` and rely on package import aliases for referencing

Reply:
I mean something like:

```python
from zenodo_rdm_migrator.actions import transform

actions = [
    transform.UserRegistration,
    transform.UserEdit,
    ...
]

# or
from zenodo_rdm_migrator.actions import transform as actions_transform

# kind of unnecessary though because we're defining this "actions" list, which already
# hints towards what it contains :)
actions = [  
    actions_transform.UserRegistration,
    actions_transform.UserEdit,
    ...
]
```